packages/block-library/src/query-pagination/edit.js

```diff
@@ -51,6 +52,10 @@ export default function QueryPaginationEdit( {
 		template: TEMPLATE,
 		allowedBlocks: ALLOWED_BLOCKS,
 	} );
+	// Always show label text if paginationArrow is set to 'none'.
+	if ( paginationArrow === 'none' ) {
```

Comment:
We need to wrap this in a `useEffect`, because it triggers React warning right now. Also we don't need to set the attribute if is already `true`.

Reply:
Thanks! This should be addressed here: https://github.com/WordPress/gutenberg/pull/51165/commits/0d862c532dd168762a187c46976c92be4a31982d